### DISCUSSION TITLE
Dropping a specific value of an array-property

### DISCUSSION THREAD

**Discussion by utillity** [Gremlinq/ExRam.Gremlinq#854] [Q&A] [answered]
hi,

I am trying to remove a specific item from an array of a vertex-property.

In the console this statement does the trick
`g.V(['pk','id']).properties('ListOne').hasValue('V3').drop()`
but I cannot build it in Gremlinq, because there is no hasValue equivalent.

The closest I got is this:
`g.V<TestVertex>(v.Key()).SideEffect(_ => _.Properties(e => e.ListOne).Value().Where(e => e == "V3").Drop())`
which resolves to
`g.V(_a).hasLabel(_b).sideEffect(__.properties(_c).value().is(_d).drop())`
but I get a runtime exception: `Gremlin query syntax error: Only objects of type vertex, edge, and property can be dropped.`

any hints would be appreciated!

cheers, Tilli

**Comment by danielcweber** [comment, marked as answer]
How about

`g.V<TestVertex>(v.Key()).SideEffect(_ => _.Properties(e => e.ListOne).Where(p => p.Value == "V3").Drop())`

**Reply by utillity** [reply]
yep, that did the trick - thanks Daniel, I appreciate it!
